orchestra/communication/tests/test_models.py

```diff
@@ -82,3 +83,57 @@ class StaffingRequestInquiryTestCase(OrchestraModelTestCase):
 class StaffingResponseTestCase(OrchestraModelTestCase):
     __test__ = True
     model = StaffingResponseFactory
+
+    def test_mark_winner(self):
+        request = StaffBotRequestFactory()
+        inquiry = StaffingRequestInquiryFactory(request=request)
+        self.assertEqual(
+            request.status,
+            StaffBotRequest.Status.SENDING_INQUIRIES.value)
+
+        # The staff bot request status should be complete when one
```

Comment:
i agree with the comment, but don't understand how the code works -> where in this test are we calling something like `mark_winner` or something like that?

Reply:
i see -> this happens in the new `save` method. see my comment on the save method asking if there's a cleaner/more explicit place in the API to put it